### ISSUE TITLE
Recirculation ads

### ISSUE THREAD

**Issue by mrgrupbg** [Automattic/fb-instant-articles#763]
# Steps required to reproduce the problem

1. Publish article

# Expected Result

*  Instant article with  recirculation ads at footer

# Actual Result

* No recirculation ads

---

The problem is that it appears nobody knows how to add simple code inside wordpress to make recirculation ads automatically inserted inside the articles. - Right now the only working solution is to add the code manually on every single instant article.

**Comment by mrgrupbg** [comment]
Anyone?!

**Comment by timjacobi** [comment]
Could you please be a little bit more specific about the input. What part of your post would you like to have converted to recirculation ads? How do you do it manually?

**Comment by mrgrupbg** [comment]
Hello ,

Right now this code circulate around the web ( filter to insert inside wp file functions.php )
This is the code that suppose to make your recirculation ads inserted automatically on every instant article you publish. 

`add_filter( 'instant_articles_transformed_element','m_add_recirculation_ad' );
function m_add_recirculation_ad ($instant_article) {
$instant_article->addMetaProperty('fb:op-recirculation-ads', 'placement_id=YOUR_PLACEMENT_ID');
return $instant_article;
}`

But the problem is that this code is not working. Nothing happen when applied. 
Instead the only working solution is to add manually this
`<meta property="fb:op-recirculation-ads" content="placement_id=YOUR-PLACEMENT-ID">
`
inside an instant article markup ( located in your fb page> publishing tools> instant articles> production articles)

So my original question was can someone help me out with making recirculation ad units visible on every instant article to avoid add the above code on every single article manually.

**Comment by timjacobi** [comment]
Thank you for clarifying this. Sorry to hear you're having a bad time getting this integrated. We don't currently support this but I will add it to the list of feature requests and we will prioritise it next time we triage the list.

**Comment by mrgrupbg** [comment]
Thanks a lot! All the best! :)

**Comment by pestevez** [comment]
Hey @mrgrupbg - we are still going to take a look at this feature request, but if your goal is to add the recirculation units to ALL articles, then we have good news: [Instant Articles Introduces “Monetization Tab”, With Simpler, One-Click Control for Turning on Recirculation Ads](https://media.fb.com/2017/11/06/instant-articles-introduces-monetization-tab-with-simpler-one-click-control-for-turning-on-recirculation-ads/)

Basically, you don't need to modify the Instant Articles markup to get the recirculation units.

**Comment by webbasica** [comment]
I had the filter enabled and after this last update I started getting this error:
`Wrong recirculation ads placement id: Please check if you set placement_id correctly for recirculation ads.`

I disabled the filter on the functions.php file, we'll see of that fixes it.
